content/client-server-api/modules/receipts.md

```diff
@@ -137,16 +137,22 @@ either a thread root's event ID or `main` for the main timeline.
 
 Threading introduces a concept of multiple conversations being held in the same
 room and thus deserve their own read receipts and notification counts. An event is
-considered to be "in a thread" if it meets any of the following criteria:
-* It has a `rel_type` of `m.thread`.
-* It has child events with a `rel_type` of `m.thread` (in which case it'd be the
-  thread root).
-* Following the event relationships, it has a parent event which qualifies for
-  one of the above. Implementations should not recurse infinitely, though: a
-  maximum of 3 hops is recommended to cover indirect relationships.
-
-Events not in a thread but still in the room are considered to be part of the
-"main timeline", or a special thread with an ID of `main`.
+considered to be "in a thread" if:
+
+* It has a rel_type of m.thread, or
+* It has a parent event with this rel_type, or a parent of a parent, or further
+  up the chain of relations. (Implementations should not recurse to arbitrary
+  depth: a maximum of 3 hops is recommended to cover indirect relationships.)
```

Comment:
This seems to have lost all the formatting:

```suggestion
* It has a `rel_type` of `m.thread`, or
* It has a parent event with this `rel_type`, or a parent of a parent, or further
  up the chain of relations. (Implementations should not recurse to arbitrary
  depth: a maximum of 3 hops is recommended to cover indirect relationships.)
```

Personally I find the wording of the last bullet more confusing than the previous wording. 🤷

Reply:
The crucial part is to remove "It has child events with a `rel_type` of `m.thread` (in which case it'd be the thread root)", which was incorrect.

Travis told me not to use the word "ancestor", which was how I previously tried to describe it.

I hate how it reads now, so suggestions welcome :-)

Reply:
Done in c8e39c10